src/picongpu/include/simulation_defines/param/radiationObserver.param

```diff
@@ -96,7 +96,7 @@ namespace picongpu
       const picongpu::float_64 phi(    my_index_phi   * delta_angle_phi   - angle_phi_start   );
 
       /* compute observation unit vector */
-      return vector_64( sinf(theta)*cosf(phi) , sinf(theta)*sinf(phi) , cosf(theta) ) ;
```

Comment:
please use `mat::sin()`, `math::cos()` or maybe `math::sincos(phi)`
If you need 32bit please cast your values with `precisionCast<float_X>()` before.